src/SourceGenerators/Uno.UI.SourceGenerators/Helpers/RoslynMetadataHelper.cs

```diff
@@ -272,5 +274,36 @@ IEnumerable<INamedTypeSymbol> GetTypesDerivingFrom(INamespaceSymbol ns)
 				}
 			}
 		}
+
+		public IEnumerable<INamedTypeSymbol> GetAllTypesAttributedWith(string attributeClassFullName)
+		{
+			return _getAllTypesAttributedWith(attributeClassFullName);
+		}
+
+		private IEnumerable<INamedTypeSymbol> GetAllAttributedTypes(string attributeClassFullName)
```

Comment:
```suggestion
		private IEnumerable<INamedTypeSymbol> SourceGetAllTypesAttributedWith(string attributeClassFullName)
```

per existing naming convention